metadata.rb

```diff
@@ -24,7 +24,7 @@
 chef_version     '>= 12' if respond_to?(:chef_version)
 depends          'compat_resource'
 
-gem              'oneview-sdk' if respond_to?(:gem)
+gem              'oneview-sdk', '=1.0.0' if respond_to?(:gem)
```

Comment:
@hdiomede , I'm wondering if we need this line in there at all. In [this](https://github.com/HewlettPackard/oneview-chef/blob/master/libraries/oneview_helper.rb#L19), we already do the gem install, and as an added benefit, we can set the version based on a node attribute. If we pin a version here, it might not ever get used.